### PR TITLE
Drop unused configPath field

### PR DESCRIPTION
**Pull request by sethvargo**
This field was [removed during a refactor of the Google Cloud Storage SDK](https://github.com/googleapis/nodejs-storage/commit/77fa8d9f95afbc830b57188ce0d2dfac46476d0b) and is no longer needed. Since we weren't asserting types, we never got an error that we were setting a field unnecessarily.

Closes https://github.com/google-github-actions/upload-cloud-storage/issues/254

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -16,13 +16,8 @@
...
 
-import { Storage, StorageOptions, PredefinedAcl } from '@google-cloud/storage';
-import {
-  randomFilepath,
-  inParallel,
-  toPlatformPath,
-  toPosixPath,
-} from '@google-github-actions/actions-utils';
+import { Storage, StorageOptions, PredefinedAcl, UploadOptions } from '@google-cloud/storage';
+import { inParallel, toPlatformPath, toPosixPath } from '@google-github-actions/actions-utils';
```
Comment (verbanicm) [inline src/client.ts:20]: Should we use actions/core instead since these were merged upstream??

File: src/main.ts [hunk 2]
```diff
@@ -156,11 +156,11 @@ export async function run(): Promise<void> {
       resumable: resumable,
       predefinedAcl: predefinedAcl,
 
-      onUploadObject: (source: string, destination: string, opts: Record<string, unknown>) => {
+      onUploadObject: (source: string, destination: string, opts: UploadOptions) => {
         core.info(`Uploading ${source} to gs://${destination}`);
 
         if (core.isDebug()) {
-          const data = deepClone(opts);
+          const data = deepClone(opts) as Record<string, unknown>;
```
Comment (verbanicm) [inline src/main.ts:163]: Why not keep this UploadOptions. Is there a reason?

Comment (sethvargo) [reply src/main.ts:163]: We're adding our own custom fields which aren't part of UploadOptions below (like timestamp)

File: src/client.ts (hunk 1 above)
Comment (sethvargo) [reply src/client.ts:20]: We already have the dependency on actions-utils. It could be a good followup, but it doesn't save us much.

**Review by verbanicm — APPROVED** [review]

**Pull request merged**